src/features/basic/cx-search-bar.tsx

```
@@ -0,0 +1,92 @@
+    }
+    for (const material of materials) {
+      if (
+        material.ticker.includes(searchTerm) ||
+        getMaterialName(material)?.toUpperCase().includes(searchTerm)
+      ) {
+        const optionElement = categoryOptions.get(material.category);
+        if (optionElement) {
+          optionElement.classList.remove(css.hidden);
+        }
+        const rowElement = materialRows.get(material.ticker);
+        if (rowElement && rowElement.isConnected) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)